### DISCUSSION TITLE
Question about date format for Vietnamese language

### DISCUSSION THREAD

**Discussion by DucDuyGaming** [ReactiveDrop/reactivedrop_translations#1258] [Q&A] [answered]
I am going to prepare to translate News and Mails, however I need some more information on what date format to use. Vietnamese date format typically use dd/mm/yyyy format. However, the [wiki here](https://github.com/ReactiveDrop/reactivedrop_translations/wiki/Date-formats-&-time-ranges-used) said that Vietnamese language use the American "mm/dd/yy" format instead. Also in the "[Used time ranges](https://github.com/ReactiveDrop/reactivedrop_translations/wiki/Date-formats-&-time-ranges-used#used-time-ranges)" section, the date format is yyyy-mm-dd for **all** data

The question is that which format should I really use. Is it a requirement that I use the date format as described in the wiki or in the original _english text files or [english] strings, or is it up to my own decision? 
https://en.wikipedia.org/wiki/Date_and_time_notation_in_Vietnam

**Comment by Duke-M-commits** [comment, marked as answer]
It is up to you, the article has been written by me. It just states the current situation. If you change the way the date formats are written I (or even you) will change the documentation on the wiki.

My personal opinion: yyyy-mm-dd would be optimal because it's the scientifically best way to write out our earthly date formats these days (NASA uses it, most of europe accepted it to be a standard). But the whole disunity about which century is the right one to be used motivated me to not touch it in my german version. In the end, it's a fantasy universe, sooo it should be consistent throughout one language and that's it.

If you decide to use a format including 4 digits for the year you have to settle on a century, I would recommend 2200, but as I said, it's up to you.

**Reply by DucDuyGaming** [reply]
Alright. I think I will be using the format dd/mm/yy (not dd/mm/yyyy), since I personally think the American date format can be a bit confusing for the Vietnamese people. The original, default date format in the Mails files (at least those I saw so far, as of writing I've only look into 2 files) use mm/dd/yy so I can't exactly use the optimal yyyy-mm-dd. I also decided to use yy and not yyyy to not not be bothered with settling on a century

As for yyyy-mm-dd, since you say it is optimal and "scientifically [the] best way to write out our earthly date formats these days", if I come across one in the original English strings/texts, I will not change the way it is until there is a reason to swap it to, for example, dd-mm-yyyy
